src/locale/locales/fr/messages.po

```diff
@@ -5070,11 +4710,11 @@ msgstr "Activer le menu déroulant"
 
 #: src/screens/Moderation/index.tsx:332
 msgid "Toggle to enable or disable adult content"
-msgstr ""
+msgstr "Activer ou désactiver le contenu pour adultes"
 
 #: src/view/screens/Search/Search.tsx:427
 msgid "Top"
-msgstr ""
+msgstr "Élevé"
```

Comment:
Definitively not `Élevé` (i.e. `higher ground` !?), and as it's used in search results and in contrast with `Latest` and `People`, let's assume it's `Meilleur` (as in `Best`).

```suggestion
msgstr "Meilleur"
```